networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java

```diff
@@ -131,9 +131,7 @@ public void onIncomingMessage(
   }
 
   /**
-   * TODO: Update link for Deneb
-   *
-   * <p>See <a
+   * See <a
    * href="https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/p2p-interface.md#blobssidecarsbyrange-v1">BlobsSidecarsByRange
```

Comment:
you haven't updated the link here

Reply:
that would be deleted and there is no reference to it now with the new spec, so just decided to leave it there

Reply:
oh but it is the pre-decoupling one. Never mind.